From 5010cbe111c34e601f1d88a28e0e2d5250b9ae0c Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Fri, 17 Apr 2015 18:25:51 +0800
Subject: [PATCH 0575/1594] MLK-10700 ARM: dts: enable uart3, uart5, uart6,
 enet1, enet2 wake up irq in GPC

commit 6c500622faa7814466d9fc57ced7874d84a3e10f from
git://git.freescale.com/imx/linux-2.6-imx.git

In megafix mode, enable uart3, uart5, uart6, enet1, enet2 wake up irq in GPC.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index f3f0193..994aeb8 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -609,7 +609,7 @@
 				compatible = "fsl,imx7d-gpc";
 				reg = <0x303a0000 0x10000>;
 				interrupts = <GIC_SPI 87 IRQ_TYPE_LEVEL_HIGH>;
-				fsl,mf-mix-wakeup-irq = <0x4000000 0xc00 0x0 0x0>;
+				fsl,mf-mix-wakeup-irq = <0x54010000 0xc00 0x0 0x400010>;
 				#power-domain-cells = <1>;
 				pcie-phy-supply = <&reg_1p0d>;
 			};
-- 
1.7.5.4

